lib/onlyoffice_webdriver_wrapper/webdriver.rb

```diff
@@ -750,13 +750,16 @@ def webdriver_error(exception, error_message = nil)
     end
 
     def wait_until(timeout = ::PageObject.default_page_wait, message = nil, &block)
+      tries ||= 3
       wait = Object::Selenium::WebDriver::Wait.new(timeout: timeout, message: message)
-      begin
-        wait.until(&block)
-        wait.until { execute_javascript('return document.readyState;') == 'complete' }
+      wait.until(&block)
+      wait.until { execute_javascript('return document.readyState;') == 'complete' }
+      wait.until { execute_javascript('return !!window.jQuery && window.jQuery.active;') == 0 }
       rescue Selenium::WebDriver::Error::TimeOutError
         webdriver_error("Wait until timeout: #{timeout} seconds in")
-      end
+      rescue Selenium::WebDriver::Error::StaleElementReferenceError
+        retry unless (tries -= 1).zero?
```

Comment:
add logger message, that `Selenium::WebDriver::Error::StaleElementReferenceError`. Help determine how often those error happens if we will want statistic